apps/server/db/moduls/coupons.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+const couponsScheme = new mongoose.Schema({
+    id:{
+        type:String,
+        require:true,
+    },
+    name: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  discount: {
+    type: Number,
+    required: true,
+  },
+ 
+  category: {
+    type: String,
+    enum: ["RFashion", "Beauty", "Home", "Food"],
+    required: true,
+  },
+  code: {
+    type: String
+  },
+  expired: {
+    type: String
+  },
+});
+const Coupon = mongoose.model("coupons", couponsScheme);
+
+exports.Coupon = Coupon;
```

Comment:
prefer not to export default because it create new instance every import 
```suggestion

```

Reply:
its fine ?